worker/src/main/scala/org/apache/celeborn/service/deploy/worker/Worker.scala

```diff
@@ -370,6 +370,9 @@ private[celeborn] class Worker(
   workerSource.addGauge(WorkerSource.SORT_MEMORY) { () =>
     memoryManager.getSortMemoryCounter.get()
   }
+  workerSource.addGauge(WorkerSource.PENDING_SORT_TASKS) { () =>
```

Comment:
would you like to add this metrics into grafana dashboard? https://github.com/apache/celeborn/blob/main/assets/grafana/celeborn-dashboard.json

Reply:
added ~